Projects/UOContent/Accounting/AccountHandler.cs

```diff
@@ -319,7 +323,7 @@ private static Account CreateAccount(NetState state, string un, string pw)
                 return null;
             }
 
-            Console.WriteLine("Login: {0}: Creating new account '{1}'", state, un);
+            logger.Information("Login: {0}: Creating new account '{1}'", state, un);
```

Comment:
This is an incorrect usage of Serilog and the Key Value Pair paradigm. Rename the parameters from {0}, {1}, {n} to actual parameters such as ``AccountName``, ``NumberOfAccounts```, etc.

Reply:
Yeah I was thinking about whether to flag this or just fix it myself in a subsequent PR. Are you thinking about this:
```cs
logger.Information("Login: {state}: Creating new account '{un}'");
```

Or something else? I am new to Serilog, so a concrete example would be great. Thanks!

Reply:
`logger.Information("Login: {State}: Creating new account {Username}", state, un)`

https://github.com/serilog/serilog/wiki/Structured-Data

Note we are only using a hardcoded console sink at the moment so it won't have any effect

Reply:
Network, commands, player speech, etc can be handled by different loggers, of which we can sink to files appropriately.
Let's handle that in another iteration.

Reply:
@karolswdev I'm aware of that, but as @immaciek mentioned, I'm trying to tackle the problem iteratively. I can handle the parameter names in another iteration.